Gemfile.lock

```diff
@@ -269,6 +270,7 @@ PLATFORMS
 
```

Comment:
![image](https://github.com/codeWithHyder/blog-rails/assets/93039359/2afa8936-dc3d-4439-88dd-adb7a178b48a)

- [ ] Great work so far but I guess you broke your migrations files because right now I am not able to complete the migrations which means I cannot proceed further with your app :thinking: Kindly fix this migrations issue and submit your project again for a review :+1: